### PR TITLE
feat: wait broadcasted data to be available from queries

### PR DESCRIPTION
**Pull request by shuplenkov**
## Issue being fixed or feature implemented

The current ack mechanism ensures only that data is committed and does not wait until the next block will be committed and data become available in DAPI (https://github.com/dashevo/js-drive/pull/429).

## What was done?

wait broadcasted data to be available from queries

## How Has This Been Tested?

With mn-bootstrap

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: src/SDK/Client/Platform/methods/contracts/broadcast.ts [hunk 1]
```diff
@@ -16,5 +17,13 @@ export default async function broadcast(this: Platform, dataContract: any, ident
 
     await broadcastStateTransition(this, dataContractCreateTransition, identity);
 
+    // Wait some time for propagation
```
Comment (shumkov) [inline src/SDK/Client/Platform/methods/contracts/broadcast.ts:20]: Let's wait for 6 seconds (min block time) before the first try.

**Comment by thephez** [comment]
This will be helpful for some things (I can eliminate a dirty hack from something I've written). But it also seems that this waiting should be optional. Some use cases may want to just store documents that are accessed infrequently (e.g. logging values to run a report once per month) and don't care whether it takes 3 seconds or 3 minutes for the data to be available. Wouldn't this PR slow down every broadcast?

**Comment by shumkov** [comment]
@thephez it doesn't slow down broadcasting, it just resolves promise when your data is actually committed to the Platform state - acknowledgment. If you don't care you can just do not use `await` or handle errors with `.catch`. We gonna rework this part next year to make it more efficient and we will introduce different levels of acknowledgments like in other distributed databases. So you will be able to send and forget, get acknowledgment from several nodes, and so on.

**Comment by thephez** [comment]
Cool, sounds good :+1:

**Review by jawid-h — APPROVED** [review]

**Pull request merged**